tests/test_activemq.py

```diff
@@ -0,0 +1,75 @@
+# stdlib
```

Comment:
When you rebase, could you also rename this file `test_activemq_xml.py` ? `rake test:coverage` only works if the convention `checks.d/x.py` -> `tests/checks/integration/test_x.py` is applied.